app/models/project_instance.rb

```diff
@@ -33,6 +35,8 @@ def self.to_csv(options = {})
     end
   end
 
-  accepts_nested_attributes_for :comments, :reject_if => lambda { |a| a[:content].blank? }
+  def info
```

Comment:
I think this is a good fit for a view helper - the models should have nothing to do with html in an ideal world